db/sql/2_seattle.sql

```diff
@@ -12,7 +12,7 @@ CREATE TABLE IF NOT EXISTS seattle_officers (
 );
 
 COPY seattle_officers (badge,full_name,title,unit,unit_description,first_name,middle_name,last_name,date)
-FROM '/seed/SPD_Roster_1-28-21.csv' DELIMITER ',' CSV HEADER;
+FROM '/seed/SPD_Rosters_Historical.csv' DELIMITER ',' CSV HEADER;
```

Comment:
Just to confirm, this will still only return info from the most recent roster, correct?

Reply:
Update: this has been verified as the case for the current search implementation